README.md

```diff
@@ -28,7 +28,7 @@ Come on in, the water's lovely. More help? Ping `jfire` or `bhousel` on
 
 ## Prerequisites
 
-* [Node.js](http://nodejs.org/) version 4 or newer
+* [Node.js](http://nodejs.org/) version 4 or newer, ([npm](https://www.npmjs.com/package/npm) version 3 or newer)
```

Comment:
Yeah, I dont understand why the npm 3 requirement was added.  node 4 / npm 2 should be fine.  I set Travis to node 5 for reasons unknown, so let me put it back to 4 and confirm..